dateparser/date.py

```diff
@@ -461,6 +467,18 @@ def date_strings():
                     if self._is_applicable_locale(locale, s):
                         yield locale
 
+        if self.detect_languages_func:
+            detected_languages = self.detect_languages_func(
+                date_string, confidence_threshold=self._settings.LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD
```

Comment:
```suggestion
                text=date_string, confidence_threshold=self._settings.LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD
```
I would use the `text` parameter here, in that way, if they implement their own `detect_languages_func` without the `text` parameter it will raise:

`TypeError: my_detect_lang_func() got an unexpected keyword argument 'text'`

And it will be easier to understand and fix.